backend/FwLite/LcmCrdt/CrdtProjectsService.cs

```diff
@@ -103,6 +103,16 @@ public async Task<CrdtProject> CreateProject(CreateProjectRequest request)
         return crdtProject;
     }
 
+    public async Task DeleteProject(string name)
+    {
+        var project = GetProject(name) ?? throw new InvalidOperationException($"Project {name} found");
```

Comment:
Should that error message be "Project {name} **not** found"?

Reply:
Yes 👍